mbofui/app/js/utils.js

```diff
@@ -60,6 +60,21 @@ var leafletize = function(data, user){
     return leafletList;
 };
 
+
+var normalizeaddressLookupResults = function(data){
+  var locationList = [];
+  _.each(data, function(location){
+    locationList.push({
+      locationName: location.display_name || location.formatted_address,
+      lat:location.lat || location.geometry.location.lat,
+      lng:location.lon || location.geometry.location.lng
```

Comment:
I feel like if lat:location.lat then lng:location.long should be lon:location.lon - thoughts?